src/components/plots/areaPlot.ts

```diff
@@ -141,6 +168,22 @@ export module Plots {
 
       return { data: allPlotData.data, pixelPoints: allPlotData.pixelPoints, selection: d3.selectAll(allElements) };
     }
+
+    protected _constructAreaProjector(xProjector: _Projector, yProjector: _Projector, y0Projector: _Projector) {
+      var definedProjector = (d: any, i: number, dataset: Dataset) => {
+        var positionX = Plot._scaledAccessor(this.x())(d, i, dataset);
+        var positionY = Plot._scaledAccessor(this.y())(d, i, dataset);
+        return positionX != null && positionX === positionX &&
+               positionY != null && positionY === positionY;
+      };
+      return (datum: any, index: number, dataset: Dataset) => {
```

Comment:
It would not be technically consistent, but that said, this should compile and it would be a bit more explanatory.  Note sure